### PR TITLE
Flow over circle

### PR DESCRIPTION
**Pull request by AlbertJP7**
## Reason
Can be used to verify flow over sphere results in a finite volume setting. Refs. #21133

## Design
Flow over circle, the 2D design of flow over sphere.

## Impact
Depicts vortex shedding in laminar flow in a 2D mesh, outputs Strouhal numbers.

### CODE REVIEW

**Comment by lindsayad** [comment]
@grmnptr I am going to assign you as the lead on this because I will be out of town for the next two and a half weeks

**Comment by grmnptr** [comment]
Okay, I will do this next Monday!

File: modules/navier_stokes/examples/flow_over_sphere_fv.i [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    prop_values = '${rho}  ${mu}'
+  [../]
+[]
+
+[Preconditioning]
+  [./SMP] #What is PJFNK
+    type = SMP
+    full = true
+    solve_type = 'NEWTON'
+  [../]
```
Comment (dschwen) [inline modules/navier_stokes/examples/flow_over_sphere_fv.i:175]:
```suggestion
  [SMP]
    type = SMP
    full = true
    solve_type = 'NEWTON'
  []
```

Comment (GiudGiud) [reply modules/navier_stokes/examples/flow_over_sphere_fv.i:175]: documentation for PJFNK is here
https://mooseframework.inl.gov/source/executioners/Steady.html

Comment (lindsayad) [reply modules/navier_stokes/examples/flow_over_sphere_fv.i:175]: A NEWTON solve type automatically sets a full preconditioner, so putting `solve_type = NEWTON` in the `Executioner` allows removal of the `Preconditioning` block entirely

**Comment by grmnptr** [comment]
@GiudGiud could you give us an independent review whenever you have some time?

File: modules/navier_stokes/examples/flow-over-circle/flow_over_circle.i [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+#
+# @incollection{schafer1996benchmark,
+#   title={Benchmark computations of laminar flow around a cylinder},
+#   author={Sch{\"a}fer, Michael and Turek, Stefan and Durst, Franz and Krause, Egon and Rannacher, Rolf},
+#   booktitle={Flow simulation with high-performance computers II},
+#   pages={547--566},
+#   year={1996},
+#   publisher={Springer}
+# }
+# The reyndols number is Re=100.
```
Comment (GiudGiud) [inline modules/navier_stokes/examples/flow-over-circle/flow_over_circle.i:14]:
```suggestion
# The Reynolds number is Re=100.
```

File: modules/navier_stokes/examples/flow-over-circle/flow_over_circle-action.i [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+#
+# @incollection{schafer1996benchmark,
+#   title={Benchmark computations of laminar flow around a cylinder},
+#   author={Sch{\"a}fer, Michael and Turek, Stefan and Durst, Franz and Krause, Egon and Rannacher, Rolf},
+#   booktitle={Flow simulation with high-performance computers II},
+#   pages={547--566},
+#   year={1996},
+#   publisher={Springer}
+# }
+# The reyndols number is Re=100.
```
Comment (GiudGiud) [inline modules/navier_stokes/examples/flow-over-circle/flow_over_circle-action.i:14]: same

**Review by GiudGiud — CHANGES_REQUESTED** [review]

File: modules/navier_stokes/examples/flow-over-circle/flow_over_circle-action.i [hunk 4]
```diff
@@ -0,0 +1,325 @@
+# -----------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (grmnptr) [reply modules/navier_stokes/examples/flow-over-circle/flow_over_circle-action.i]: Separated it to multiple inputs!

**Review by GiudGiud — APPROVED** [review]
Thanks for the excellent work @AlbertJP7 and the refactor @grmnptr !
This is more evidence that the module is working as desired #validation

**Pull request merged**